README.md

```diff
@@ -43,3 +59,20 @@ Follow the [./PREREQUISITES.md](./PREREQUISITES.md) instructions to configure a
     * http://localhost:8080/info
 * You can access the MongoDB Admin UI at:
   * http://localhost:8090 
+
+---
+
+## Code Conventions
+
+### Solving tickets
+
+* a branch starting off from `main` should be created for each ticket from the [Project Board](https://github.com/orgs/PascalTeam/projects/1/views/1)
+* the branch name should follow this pattern: `<type>/<ticketNumber>/<shortDescription>`
+  where:
+    * **type** can be one of the following: _fix_(bug), _feat_(feature), _docs_(documentation-related changes)
+    * **ticketNumber**: the issues number associated with the ticket from the board
+    * **shortDescription**: a short description of the ticket
+
+  _Most of these ideas came from [this document](https://gist.github.com/brianclements/841ea7bffdb01346392c)._
```

Comment:
Maybe it would be worth adding requirements like "each PR must be approved by at least one person"?

Reply:
Sure, will add that.